main.star

```diff
@@ -1,40 +1,50 @@
-MONGO_DB_PROTOCOL_NAME = "mongodb"
-MONGO_DB_PORT_ID = "mongodb"
```

Comment:
I intentionally removed this because I would prefer people not need to think about configuring port numbers (one of the value props of Kurtosis)